lib/src/base/audio_waveforms_interface.dart

```diff
@@ -122,6 +122,12 @@ class AudioWaveformsInterface {
     return result ?? false;
   }
 
+  ///platform call to release resource
+  Future<bool> release(String key)async{
+    var result = await  _methodChannel.invokeMethod(Constants.releaseAudioPlayerResource,{Constants.playerKey : key});
```

Comment:
This looks unformatted. Add comma after `key` and `}` so that it will be formatted correctly.